contributing.md

```diff
@@ -0,0 +1,122 @@
+# Contributing Quickstart
+
+## Purpose
+
+This guide will help you quickly set up a development environment where you can contribute code to Handy. This guide will be especially useful for newcomers to Rails development.
+
+Specifically, this guide will show you how to use VirtualBox and Vagrant to set up your dev environment with a virtual machine (VM). If you feel comfortable with Rails and don't want to use a VM, this guide isn't for you.
+
+## Prerequisites
+
+We'll use VirtualBox and Vagrant to prepare our environment.
+
+Go [here](https://www.virtualbox.org/) to get VirtualBox for your machine.
+
+Then, go [here](https://www.vagrantup.com/) to get Vagrant.
+
+VirtualBox is a VM provider. It does the actual work of hosting a fake computer (the VM) in your computer (the host). Vagrant is a tool that works on top of VirtualBox. It makes sharing and configuring development environments easy.
+
+Once you've installed the tools, I recommend going through Vagrant's [_Getting Started_](http://docs.vagrantup.com/v2/getting-started/index.html) guide so you understand what's going on and you're confident your installations were successful. Or don't. I'm not the boss of you.
+
+## Get the Quickstart Box
+
+I created a "base box" configured specially for Handy development. It's an Ubuntu (Linux) image with all the dependencies you'll need to start the development server*. You can download it from my Google Drive [here](https://drive.google.com/file/d/0B4TO8N-e7-6YbDhBZXROY19xY0k/view?usp=sharing). 
+
+Once you've downloaded handy-quickstart.box, let Vagrant know it exists like so:
+`vagrant box add handy-quickstart handy-quickstart.box`
+
+## Configure the VM
+
+Now you're ready to spin up your virtual machine. In the directory where you want to place Handy's code, make a file called `Vagrantfile`. It must be named `Vagrantfile`. It must have no extension. (e.g. `Vagrant.txt` will fail.)
+
+The Vagrantfile is where we will configure our VM. Below is the Vagrantfile I use. I recommend you use it as-is (assuming you have at least 8 GB of RAM and 2 processors).
+
+```
+Vagrant.configure(2) do |config|
+
+  config.vm.box = "handy-quickstart"
+  config.vm.network :forwarded_port, guest: 3000, host: 3000
+  
+  config.vm.provider "virtualbox" do |v|
+      v.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
+      v.customize ["modifyvm", :id, "--natdnsproxy1", "on"]
+      v.memory = 4096
+      v.cpus = 2
+  end
+
+end
+```
```

Comment:
I didn't include it in the repo because I thought if you and Joel weren't using Vagrant, then why add another file. But if you're for it, then there's no reason not to.